app/editor/src/components/navbar/NavBarItem.tsx

```diff
@@ -1,10 +1,24 @@
+import { useNavigate } from 'react-router';
 import * as styled from './NavBarItemStyled';
 
 export interface INavBarItemProps extends React.HTMLProps<HTMLButtonElement> {
-  children?: React.ReactNode;
+  /**
+   * choose the tab label
+   */
+  label?: string;
+  /**
+   * prop used to determine whether the tab is active
+   */
   active?: boolean;
+  /**
+   * the path the item will navigate you to
+   */
+  navigateTo?: string;
 }
 
-export const NavBarItem: React.FC<INavBarItemProps> = ({ children, active }) => {
-  return <styled.NavBarItem>{children}</styled.NavBarItem>;
+export const NavBarItem: React.FC<INavBarItemProps> = ({ children, label, navigateTo }) => {
```

Comment:
We both need to start adding descriptive comments to our components.

Reply:
I will do that before I merge!